omnibus/files/private-chef-ctl-commands/install.rb

```diff
@@ -4,7 +4,6 @@
 KNOWN_ADDONS = %w(
   chef-ha
   chef-sync
-  opscode-analytics
```

Comment:
This was requested by support.  Will it break any of the existing demo frameworks?